File: label/python/gtk/label/label.py
#! /usr/bin/env python3

import gi
gi.require_version('Gtk', '3.0')
from gi.repository import Gtk

window = Gtk.Window()
window.set_title("Label")
label = Gtk.Label.new("Hello, world!")
window.add(label)
window.connect("delete-event", Gtk.main_quit)
window.show_all()
Gtk.main()

File: label/javascript/gtk/label/label.js
imports.gi.versions.Gtk = "3.0";
const Gtk = imports.gi.Gtk;

Gtk.init(null)
let win = new Gtk.Window();
win.set_title("Label");
let label = new Gtk.Label({label: "Hello, world!"});
win.add(label);
win.connect("delete-event", () => Gtk.main_quit());
win.show_all()
Gtk.main()
